hikari/events/__init__.pyi

```diff
@@ -0,0 +1,12 @@
+from hikari.events.channel_events import *
+from hikari.events.guild_events import *
+from hikari.events.lifetime_events import *
+from hikari.events.member_events import *
+from hikari.events.message_events import *
+from hikari.events.reaction_events import *
+from hikari.events.role_events import *
+from hikari.events.shard_events import *
+from hikari.events.typing_events import *
+from hikari.events.user_events import *
+from hikari.events.voice_events import *
+from hikari.events.base_events import Event as Event, ExceptionEvent as ExceptionEvent
```

Comment:
How did you manage to add these :sweat_smile:. I thought they were in the `.gitignore`. Could you please remove them?

Reply:
lmao